### ISSUE TITLE
Identicons is causing a hook call warning

### ISSUE THREAD

**Issue by Tobio89** [tuhnik/react-identicons#15]
```
Error: Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:
1. You might have mismatching versions of React and the renderer (such as React DOM)
2. You might be breaking the Rules of Hooks
3. You might have more than one copy of React in the same app
See https://fb.me/react-invalid-hook-call for tips about how to debug and fix this problem.
```
I've got this issue when I try to include identicons in my project.
The project is here: https://github.com/BruceJi7/websocket-chat-app/tree/identicon-issue

I used identicons in another project successfully. I'm not sure why this issue is happening - the error message flags a different part of the code as being wrong, but the code works if identicons aren't used.

**Comment by Tobio89** [comment]
Further digging suggests the issue may be that I'm using React 17.0.1 and identicons requires 16.13.0 / 16.14.0

**Comment by booleanBoy** [comment]
Same issue, also on React 17.0.1.

**Comment by doke-v** [comment]
Should be fixed in v1.2.5.

**Comment by Tobio89** [comment]
Rad, I'll install it tomorrow and give it another go. Thanks!
Edit: Thanks again, it works great
